sdk/keyvault/azure-security-keyvault-certificates/README.md

```diff
@@ -182,7 +182,7 @@ System.out.printf("Certificate is returned with name %s and secret id %s \n", ce
 Retrieve a previously stored Certificate by calling `getCertificate` or `getCertificateWithPolicy`.
 ```Java
 Certificate certificate = certificateClient.getCertificateWithPolicy("certificateName");
```

Comment:
Need to update `.endpoint` in the builder to `.vaultUrl` ,[here](https://github.com/Azure/azure-sdk-for-java/pull/5910/files#diff-824e08966bb186e6f79351c1a042d490L130)

Reply:
Certificate changes not part of this PR. separate PR.